### ISSUE TITLE
Custom esign page if next friend

### ISSUE THREAD

**Issue by normon66** [mplp/docassemble-MLHPPOAndProposedOrder#115]
If the petitioner has a next friend, the next friend is asked early on if **they** want to esign forms. 

The esign page in the outro refers to the petitioner (ie the minor or incapacitated adult) esigning forms. In this scenario, it would be helpful to include the petitioner's name to clarify which signature is being sought. 

![image](https://github.com/user-attachments/assets/aa5ad124-805f-41e3-8d5b-2521df329be7)

![image](https://github.com/user-attachments/assets/db19f3f1-2cc1-4ca6-adc9-70fbf2efbe13)

**Comment by RamiLorca** [comment]
@normon66 @ekressmiller - Since this signature question is being pulled from the MLH framework, would it make sense to make the edit there?

**Comment by ekressmiller** [comment]
I don't think we should ask twice if the next friend wants to e-sign. (I think maybe I had a separate issue about this, but it looks like it's already been closed.) 

Assuming we get rid of the first question about e-signing and just ask here at the end if the next friend wants to e-sign, my instinct would be to just copy the code in from the framework and edit it in this interview as needed. The next friend thing is relatively rare, I think. But open to other takes.

**Comment by ekressmiller** [comment]
Or maybe I'm misunderstanding? If there's a next friend, I thought _only_ the next friend was supposed to sign the forms. Is that inaccurate?

**Comment by normon66** [comment]
Sorry I'm late to the game. According to the Request for Next Friend (MC 319), there are times when both the next friend AND the person needing the next friend need to sign, per the highlighted note:
   ![image](https://github.com/user-attachments/assets/43e79894-cb07-482a-8885-c49c2c60ba01)

Upon reading this, I think I actually would need to adjust the A2J interview a bit. Right now, it asks for the e-signature of the petitioner, even if they're a minor 14 or an incapacitated adult.

However, in the A2J interview, I do skip the minor's esign page if the next friend didn't want to esign since they'd need to print and hand-sign the forms, anyway.
   ![image](https://github.com/user-attachments/assets/e0bd6fb9-d702-400a-b1a2-2673760b2a80)
   ![image](https://github.com/user-attachments/assets/457a553a-aeda-4b62-85d3-a0a6f1df87c8)

Feel free to reach out if I can provide any additional clarity or if you have different interpretations. 

@ekressmiller @RamiLorca

**Comment by ekressmiller** [comment]
Thanks for all of that @normon66!

@RamiLorca, with the additional info from Brett, obviously it looks like there's some more logic to build in. But going back to the original question, I would suggest you customize a screen asking for the petitioner's e-signature when appropriate in next friend situations. I think we want to be unusually clear about who is signing when there's a next friend.

I think (assuming it wouldn't cause other logic problems) it probably would make sense to move the next friend e-signing question to the end and do it either right before, or possibly even on the same screen, where we ask for the _petitioner_ signature (when appropriate). Can also discuss at the meeting this week if you'd like!
